### PR TITLE
[ci] Add placeholder script and target

### PR DESCRIPTION
**Pull request by aravindhp**
- Add a Makefile target that will end up getting called by the CI job for running our e2e tests
- Add a placeholder script that will kick off the e2e tests

### CODE REVIEW

File: hack/run-ci-e2e-test.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# TODO: Add steps to execute code added in WINC-65
```
Comment (sebsoto) [inline hack/run-ci-e2e-test.sh]: Do we want to mention ticket names here? Only because this is public and they are private.

Comment (aravindhp) [reply hack/run-ci-e2e-test.sh]: I can remove it.

**Comment by aravindhp** [comment]
@sebsoto I have removed the Jira link. PTAL.

**Comment by ravisantoshgudimetla** [comment]
/lgtm 

Once more..

**Pull request merged**